complaints/ccdb/verification/cache_prev_size.txt

```diff
@@ -0,0 +1 @@
+0
```

Comment:
I assume these files are updated at the end of the `make s3` target

Reply:
These files are updated at the end of the verification process, and then the files are stored in the Jenkins workspace to be used in the next verification run. That is the basic functionality of the index-check jobs that I extended for verification